### DISCUSSION TITLE
How to get 0.2.8 duckdb_jdbc?

### DISCUSSION THREAD

**Discussion by yxmm-wxe** [duckdb/duckdb#2141] [Q&A] [answered]
0.2.8 duckdb has been released, but I cannot get 0.2.8 duckdb_jdbc from [Maven Central](https://mvnrepository.com/artifact/org.duckdb/duckdb_jdbc)

How to get 0.2.8 duckdb_jdbc?

**Comment by hannes** [comment, marked as answer]
We still have to upload that version, stand by
